### DISCUSSION TITLE
'Birdseye' page open at "full Screen"

### DISCUSSION THREAD

**Discussion by Oddsodz** [blakeblackshear/frigate#9323] [Ask A Question]
Hello folks.

I have a query about having the 'Birdseye' page open at "full Screen"

What my use case is for is to have a little older computer running ubuntu.  And have it so that once it powers on. It loads up and auto login to the desktop.  Then automatically open up the http://notyoulocaldomainname.com:5000/birdseye

This far I have got.  And it is working nice.

But the cherry on top would be if there was a way to have it open that page but in 'full screen'

Something like this below.

http://notyoulocaldomainname.com:5000/birdseye=fullscreen

Anyway.  Hope somebody can help with a possible workaround

Thanks for reading.

Oddsodz

**Comment by DakotaS96** [comment]
I agree. I'd really like to know if this feature exists or is planned to come in the future. I have several Raspberry Pi's that I like to run as a kiosk with Chromium opening directly to a web link on bootup. It would be great if this link can open full screen for Birdseye.

**Comment by ColeCoffman** [comment]
What I do is on my windows machine I use VLC to watch the Birdseye restream. It may be possible to do the same thing on ubuntu.

**Comment by moandcompany** [comment]
I'm also interested in the ability to open the birdseye page with a URL parameter to expand width, but using Birdseye with the `restream` option as mentioned works well for remote viewing/kiosk use cases. There are also Raspberry Pi oriented packages for a kiosk-style remote camera viewer. [Example](https://github.com/SvenVD/rpisurv)

[Restream option](https://docs.frigate.video/configuration/restream)

You can use VLC to open, via RTSP, a copy of the birdseye stream through `go2rtc`.
[Restream Birdseye with Frigate+go2rtc](https://docs.frigate.video/configuration/restream#birdseye-restreaml)

**Comment by akayami-legacy** [comment]
The restream is a good option, but requires an rtsp player. It would however be useful to be able to make the webview go into fullscreen automatically width some URL parameters as there are situation where playing though browser would be ideal.

**Comment by cupertinomiranda** [comment]
This feature would really be useful to allow the birdseye view to be used as screensaver in Fully Kiosk Browser, allowing the brightness to be dimmed and still have overview of what is happening outside.

**Comment by goofballtech** [comment]
Adding another voice to this chorus. I have the TV's in my house set to use a URL as a screensaver. My curent instance of BlueIris allows paramaetr setting via URL and its very nice to have the camera just exist on any idle screens around the house.

**Comment by hawkeye217** [comment]
Browsers prevent pages from automatically opening a page in full screen view as a security measure. This is not something Frigate can work around.

If you can mimic a keystroke somehow, the "F" key toggles the fullscreen feature in Frigate 0.14 and later.

**Reply by goofballtech** [reply]
and yet... The below is on every idle screen in my house. Screen seems pretty full to me. I just [made a screensaver](https://www.finalhit.com/products/live-screensaver-creator/) from the BlueIris URL with the full screen param setup. Pretty sure the screensaver app hides all its own controls so all you have to do is hide all your own controls. 

![20250624_103746](https://github.com/user-attachments/assets/a310c776-bc05-41ce-8912-833c9be0af59)
